examples/default/realign.json

```diff
@@ -13,48 +13,79 @@
     },
   "@graph": [
     {
+      "prov:SoftwareAgent" : [
+        {
+          "@id" : "653467354367537gjsfhdgfhfdsjkn",
+          "@type" : "prov:SoftwareAgent",
+          "RRID" : "https://scicrunch.org/resources/Any/record/nlx_144509-1/SCR_007037/resolver?q=SPM&l=SPM",
+          "label" : "SPM" 
+        }
+      ],
       "prov:Activity" : [
         {
           "@id" : "realign",
```

Comment:
To be closer to a real-world example, can you use an alphanumeric id (uuid) and keep the name in a "label" attribute?